registry-requirements.md

```diff
@@ -122,3 +69,48 @@ Same content as existing binding
 (Think about how to manage the set of semantic artifacts/ontologies)
 
 Create a repository template to make it easy for people to create new bindings
+
+## Entry Requirements
+
+
+## Basic Requirements
+
+To choose a mechanism and the rules, we document our expectations below:
+
+- A binding SHOULD be written by people with a good understanding of the protocol, media type (or similar), and not necessarily the TD Editors. This includes people and organizations inside and outside of the WoT WG.
+  - Reasons Behind the Requirement:
+    - Instead of WG learning each new protocol and media type, it is more efficient for people with a good understanding of the protocol or media type to write a binding.
+    - Engaging other communities.
+- A binding SHOULD correspond to specific TD specification version(s).
+  - Reason Behind the Requirement: A binding may not fit newer or older versions of a TD specification (e.g., `readproperty` can become `readprop`, or a new operation can arrive). Thus, at the time of writing a binding, it needs to be associated with one or more known TD specification versions.
+- The binding registry MUST be a separate document but associated with a TD version.
+  - Reasons Behind the Requirement: It is easier to update in the long term.
+- Association of a binding with the TD specification (registry entry) SHOULD be confirmed by the WoT Working Group. In other words, a person needs some permission and/or confirmation to authoritatively say that a given binding can be used with TD version X. The custodian of this registry is the WoT WG.
```

Comment:
It would be better to make a registry entry require that some test(suite) be passed, making the requirement clearly technical. As written, politics could come into play, to the detriment of all involved.

Reply:
Added a DISCUSS comment about this